src/main/resources/application.yml

```diff
@@ -68,6 +68,21 @@ courtfinder:
   api:
     url: "https://courttribunalfinder.service.gov.uk"
 
+fees:
+  api:
+    url: "http://localhost:4182"
```

Comment:
These'll need to come from a var, and a `cmc-integration-tests` PR for local envs, right?

Reply:
yeah should get them from env variables 👍